### PR TITLE
Extend mission schema

### PR DESCRIPTION
**Pull request by Iliasgx**
- Made missions separate files instead of one array.
- Extended the schema with new data findings.

### CODE REVIEW

File: examples/missions/example.json [hunk 1]
```diff
@@ -1,91 +1,61 @@
...
             "rarity": "uncommon",
-            "value": 50
+            "value": 18
           }
         ],
         "others": [
           {
-            "type": "federation credits",
-            "value": 330
+            "type": "romulan points",
```
Comment (WebSpider) [inline examples/missions/example.json:33]: The correct name is 'reputation'

**Review by WebSpider — COMMENTED** [review]
Looks good, just one remark on the reputation.

File: examples/missions/example.json (hunk 1 above)
Comment (Iliasgx) [reply examples/missions/example.json:33]: The info was based on lcars data. 'Reputation' is the goal you reach like Respected and Ally. The points itself are called Faction Reputation Points, in game they also use the 'points': https://cdn.discordapp.com/attachments/869657307955490817/886591521435041822/Screenshot_20210912_143717_com.scopely.startrek.jpg

Comment (WebSpider) [reply examples/missions/example.json:33]: Ok, this has been retranslated! 👍

**Review by WebSpider — APPROVED** [review]

**Pull request merged**